### DISCUSSION TITLE
Exclude default values from serialization

### DISCUSSION THREAD

**Discussion by dougmeredith** [charleskorn/kaml#498] [General]
With JSON serialization we have control over whether or not to serialize default vaules, and the default is not to do so. Kaml enclodes them by default. Is there any way to change this?

**Comment by charleskorn** [comment]
Have you tried setting `YamlConfiguration.encodeDefaults` to `false`?

**Reply by dougmeredith** [reply]
Sorry, I see you replied promptly, but somehow I didn't see this message until now.  Thank you, I'll give this a try, and thank you for this awesome project.
